src/main/java/net/greghaines/jesque/worker/WorkerImpl.java

```diff
@@ -615,7 +618,8 @@ private void removeInFlight(final String curQueue) {
         if (SHUTDOWN_IMMEDIATE.equals(this.state.get())) {
             lpoplpush(key(INFLIGHT, this.name, curQueue), key(QUEUE, curQueue));
         } else {
-            this.jedis.lpop(key(INFLIGHT, this.name, curQueue));
+            this.jedis.evalsha(this.removeInFlightHash.get(), 2, key(INFLIGHT, this.name, curQueue),key("log"));
```

Comment:
Referencing instance variable thru **_this_** is the convention in this project, there are few other occurrences of **_this.jedis_** in this class. Why should we remove this exact reference that even was not introduced by us?